src/app/shared/components/paragraph-oe-banner/paragraph-oe-banner.component.ts

```diff
@@ -16,6 +16,7 @@ export class ParagraphOeBannerComponent {
   @Input() credit: string;
   @Input() oeParagraphsVariant: BannerVariant;
```

Comment:
this is what i don't understand, we know that our banner component will fetch data from a drupal field called oeParagraphsVariant but for us this is just, simply, "variant", why would we match the names (complex ones) of the drupal data structure? Our component could be driven by any other data source, its inputs should semantically identify  what the input is for, in a clear way. Drupal will give us data, but it's not defining our components inputs.